src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -495,6 +495,23 @@ public override void Accept(OperationVisitor visitor)
         }
     }
 
+    internal partial class BoundTupleExpression 
+    {
+        //PROTOTYPE: we need to make this its own OperationKind 
```

Comment:
I have seen either way. I think for a branch with only one feature, the difference is not very big.